src/Illuminate/Foundation/Console/ClearCompiledCommand.php

```diff
@@ -25,17 +25,22 @@ class ClearCompiledCommand extends Command {
 	 */
 	public function fire()
 	{
-		$compiledPath = $this->laravel->getCachedCompilePath();
-		$servicesPath = $this->laravel->getCachedServicesPath();
-
-		if (file_exists($compiledPath))
+		if (method_exists($this->laravel,'getCachedCompilePath'))
 		{
-			@unlink($compiledPath);
+			$compiledPath = $this->laravel->getCachedCompilePath();
```

Comment:
Put the variable assignment into the brackets in the if statement to save space.